WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -503,6 +503,12 @@
     context 'when signed in as results team member' do
       sign_in { FactoryGirl.create(:results_team) }
 
+      # Posts should always be in English, therefore we want to check using an English text,
+      # even if the user posting has a different locale
+      before :each do
```

Comment:
Just curious if we need to worry about doing any comparable session cleanup after our tests?